enterprise/frontend/src/metabase-enterprise/snippets/components/CollectionOptionsButton.jsx

```diff
@@ -74,7 +71,12 @@ export default class CollectionOptionsButton extends Component {
         onClick: () => onEdit(collection),
       });
     }
-    if (user && user.is_superuser) {
+    if (
+      user &&
+      user.is_superuser &&
+      (hasPremiumFeature("content_management") ||
```

Comment:
I'd like to keep `hasPremiumFeature` checks only to plugin initialization code. Can we move it there?

Reply:
Yup I can do that along with changing the flag to `snippet_collections`

Reply:
This check ended up being redundant (component is already gated by a plugin) so I removed it.